.github/workflows/build.yml

```diff
@@ -8,28 +8,42 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
-    container: ghcr.io/zephyrproject-rtos/ci:v0.26.2 
-    env:
-      CMAKE_PREFIX_PATH: /opt/toolchains
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           path: example-application
 
-      - name: Initialize
-        working-directory: example-application
-        run: |
-          west init -l .
-          west update -o=--depth=1 -n
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Setup Zephyr project
+        uses: zephyrproject-rtos/action-zephyr-setup@main
```

Comment:
Please tag a release (e.g. `v1`) for `action-zephyr-setup` and point this workflow to it in order to ensure that breaking changes in the upstream action does not end up breaking all downstream workflows.

Reply:
tagged and updated